Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.4
+
+# Run as non-root, eventually
+RUN groupadd -r slamon && useradd -r -d /workspace -m -g slamon slamon 
+
+# Install dependencies
+RUN pip install requests python-dateutil responses
```

Comment:
I think we can rely on pip to install all the required dependencies when installing the agent later. This way we don't need to keep redundant lists of dependencies.